.github/workflows/wheels.yml

```diff
@@ -136,16 +151,19 @@ jobs:
           # A shallow clone can work when the following issue is resolved:
           # https://github.com/actions/checkout/issues/338
           fetch-depth: 0
-      # Used to push the built wheels
-      - uses: actions/setup-python@v3
+
+      # Used to push the built sdist
+      - uses: conda-incubator/setup-miniconda@v2
         with:
-          # Build sdist on lowest supported Python
-          python-version: '3.8'
+          auto-update-conda: true
+          python-version: '3.8' # Build on lowest supported version
+
       - name: Build sdist
         run: |
           pip install build
           python -m build --sdist
       - name: Test the sdist
+        shell: bash -el {0}

```

Comment:
Not for this PR, but I think it'd be nice to have this at the beginning for all steps. I think I tried once and had some problems, even  if it's supposed to be supported. But maybe worth having a look at some point, as we're using conda environments most of the time.